### DISCUSSION TITLE
CI/CD Failure due to GITHUB TOKEN

### DISCUSSION THREAD

**Discussion by pdarshinibala** [community/community#127239] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

We have been using GITHUB_TOKEN while deploying packages to GITHUB and its suddenly failing since yesterday. After replacing GITHUB_TOKEN with PAT, it started working. So, we would like to know the reason why GITHUB_TOKEN has suddenly stopped working.

Please find below the command we used for GITHUB ACTIONS.

name: Publish to Github packages
run: "mvn deploy ${{ inputs.mvn-opts }} -e -X"
env:
GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

**Comment by gil-son** [comment]
Hello, how are you? I feel for that situation, you need to verify it

Access the [Github Support](https://support.github.com/)

Click on My Ticket -> New Ticket -> Finally describe it

If I supported you consider to mark my answer. Good Luck!
